apiary.apib

```diff
@@ -1253,3 +1253,55 @@ All custom user data is stored in the profile.
                 "detail": "'type' and 'id' provided don't match any existing document"
             }]
         }
+
+## User Session [/accounts/{id}/sessions]
+
+
+
++ Parameters
+
+    + id: abc4567 (required, string) - id of account
+
+### Create [POST]
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer sessionid123
+
++ Response 201 (application/vnd.api+json)
+
+        {
+            "links": {
+                "self": "https://example.com/accounts/abcd123/sessions/session123"
+            },
+            "data": {
+                id: 'session123',
+                account: {
```

Comment:
Looking at other blueprints, I should probably add a type. would that be `session`?